13.0/apache/entrypoint.sh

```diff
@@ -6,6 +6,11 @@ version_greater() {
     [ "$(printf '%s\n' "$@" | sort -t '.' -n -k1,1 -k2,2 -k3,3 -k4,4 | head -n 1)" != "$1" ]
 }
 
+# version_equal A B returns whether A = B
+version_equal() {
+    [ "$(printf '%s\n' "$@" | sort -t '.' -n -k1,1 -k2,2 -k3,3 -k4,4 | head -n 1)" = "$1" ]
```

Comment:
This function is simple the inverse of the function above.
So either the name is wrong or the implementation is wrong.